### PR TITLE
Feature/tet 487 migrate investment evidence upload to react router

### PR DESCRIPTION
**Pull request by Richard-Pentecost**
## Description of change
The form and upload functionality for uploading an investment project evidence document has been moved and from the express app into react. The look of the form has been slightly updated in line with govuk design system.

## Test instructions

Should be able to upload evidence document, and the form should show errors if required fields aren't filled out

### Before

image

### After
image

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to main?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

**Review by cgsunkel — APPROVED** [review]

File: src/client/modules/Investments/Projects/Evidence/tasks.js [hunk 1]
```diff
@@ -7,3 +9,41 @@ export const deleteProjectDocument = (values) => {
   }
   return apiProxyAxios(options)
 }
+
+export const addProjectDocument = async ({ projectId, values }) => {
+  const { tags, comment, file } = values
+
+  const fiveGigabytes = 5000 * 1024 * 1024
+  if (file.size > fiveGigabytes) {
+    throw 'File must be no larger than 5GB'
```
Comment (paulgain) [inline src/client/modules/Investments/Projects/Evidence/tasks.js:18]: What happens in the app when the string is thrown? I notice it's unhandled, i.e. there's no try...catch. I'm curious to know how the error is handled gracefully. I don't see a test for this either.

**Review by paulgain — COMMENTED** [review]
Looks good, just a couple of comments.

File: src/client/modules/Investments/Projects/Evidence/AddProjectDocument.jsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import urls from '../../../../../lib/urls.js'
+import FieldAddAnother from '../../../../components/Form/elements/FieldAddAnother/index.jsx'
+import { TASK_ADD_PROJECT_DOCUMENT } from './state.js'
+import { EvidenceTagResource } from '../../../../components/Resource/index.jsx'
+import ResourceOptionsField from '../../../../components/Form/elements/ResourceOptionsField/index.jsx'
+import { buildProjectBreadcrumbs } from '../../utils.js'
+import { FORM_LAYOUT } from '../../../../../common/constants.js'
+import FieldFileUpload from '../../../../components/Form/elements/FieldFileUpload/index.jsx'
+import { transformEvidenceForApi } from './transformers.js'
+import InvestmentName from '../InvestmentName.jsx'
```
[the remark this answers is not in the document]
Comment (Richard-Pentecost) [reply src/client/modules/Investments/Projects/Evidence/AddProjectDocument.jsx:21]: Thanks, removed the index.js and all file extensions to imports

File: src/client/modules/Investments/Projects/Evidence/tasks.js (hunk 1 above)
Comment (paulgain) [reply src/client/modules/Investments/Projects/Evidence/tasks.js:18]: Thanks for the explanation.

**Review by paulgain — APPROVED** [review]

**Pull request merged**